src/WCSimWCAddDarkNoise.cc

```diff
@@ -296,6 +296,7 @@ void WCSimWCAddDarkNoise::AddDarkNoiseBeforeDigi(WCSimWCDigitsCollection* WCHCPM
 	    ahit->SetPhotonEndPos(PMTindex[noise_pmt], pmt_position);
 	    ahit->SetPhotonStartDir(PMTindex[noise_pmt], -pmt_orientation);
 	    ahit->SetPhotonEndDir(PMTindex[noise_pmt], -pmt_orientation);
+      ahit->SetPhotonCreatorProcess(PMTindex[noise_pmt],kUnknownProcess);
```

Comment:
Instead of `kUnknownProcess`, it'd actually be really nice to extend `ProcessType_t` to include a `kDarkNoise`
This'll make it possible to distinguish between dark noise (something that is not a _physics_ process that is defined in Geant, but a known _PMT/electronics_ process) from a _physics_ process that isn't properly setup in Geant (basically some horrible edge case that shouldn't happen) or in `ProcessType_t` (basically something in Geant that isn't in WCSim's `ProcessType_t`)

```suggestion
      ahit->SetPhotonCreatorProcess(PMTindex[noise_pmt],kDarkNoise);
```

Reply:
I think this is nice but it will also give the primaries in every event the creator process name "Dark Noise". What do you think.

Reply:
Tom is suggesting to have an extra `kDarkNoise` in addition to `kUnknownProcess`.
But I think one can distinguish a dark noise from the track id or parent id, so I think it's not strictly necessary to have a `kDarkNoise`

Reply:
As I said I would prefer not to touch this since I found this creatorProcess thing very tricky. But if you consider it is something it needs to be changed, I'll try.

Reply:
I **think** it's just
- add a `kDarkNoise` to the enum (and enum string functions) in `WCSimEnumerations`
- use `kDarkNoise` here in `WCSimWCAddDarkNoise`
- Everywhere else can keep using `kUnknownProcess`

It's up to you if you want to implement it now. If not, I can after we merge this

@kmtsui it's just a ease of use idea, rather than something absolutely required. But since we know what it is I do prefer to use our knowledge rather than keep it *unknown*

Reply:
Oki just did. Hope it works just fine.